src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImpl.java

```diff
@@ -106,6 +112,91 @@ public ProfessionalUser findProfessionalUserByEmailAddress(String email) {
         return professionalUserRepository.findByEmailAddress(RefDataUtil.removeAllSpaces(email));
     }
 
+    @Override
+    public ResponseEntity<Object> fetchUsersForRefresh(String since, String userId, Integer size, UUID searchAfter) {
+        if (since != null) {
+            if (size != null) {
+                Pageable pageable = createPageableObject(0, size, Sort.by(Sort.DEFAULT_DIRECTION, "id"));

```

Comment:
createPageableObject only get called if both since and size are set, but the document suggest only only could be set, which is not catered for here.

Reply:
only enable pagination if both `since` & `size` provided. but if only `since` is provided, pagination is not needed & all users are returned. see `findRefreshUsers(String since, UUID searchAfter)` method